### PR TITLE
feat(recordings): window filter for events and console list

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Adds logics, components, and tests for filtering events and console logs by window id.

Most of the diffs are tests + fixtures (I swear!)

https://user-images.githubusercontent.com/13460330/185468910-f8e66e75-9fe0-46dd-9283-b476407209da.mov

https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=11995%3A59832

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Logic tests

### CODE REVIEW

**Review by clarkus — APPROVED** [review]

File: frontend/src/lib/constants.tsx [hunk 1]
```diff
@@ -123,7 +123,8 @@ export const FEATURE_FLAGS = {
     PERSON_MODAL_EXPORTS: 'person-modal-exports', // hot potato see https://github.com/PostHog/posthog/pull/10824
     BILLING_LOCK_EVERYTHING: 'billing-lock-everything', // owner @timgl
     IN_APP_PROMPTS_EXPERIMENT: 'IN_APP_PROMPTS_EXPERIMENT', // owner: @kappa90
-    SESSION_RECORDINGS_PLAYER_V3: 'session-recording-player-v3', // owner @alexkim205
+    SESSION_RECORDINGS_PLAYER_V3: 'session-recording-player-v3', // owner: @alexkim205
+    SESSION_RECORDINGS_PLAYER_V3_FILTERING: 'session-recording-player-v3-filtering', // owner: @alexkim205
```
Comment (rcmarron) [inline frontend/src/lib/constants.tsx:127]: Should this just be a part of the other feature flag?

File: frontend/src/scenes/session-recordings/__mocks__/recording_events.json [hunk 2]
```diff
@@ -63,5 +63,31 @@
...
+        "properties": {
+            "$window_id": "182830cdf4b28a9-02530f1179ed36-1c525635-384000-182830cdf4c2841"
+        },
+        "elements": [],
+        "elements_hash": ""
+    },
+    {
+        "id": "gooddog",
+        "name": "gooddog",
+        "event": "gooddog",
```
Comment (rcmarron) [inline frontend/src/scenes/session-recordings/__mocks__/recording_events.json:83]: lol

File: frontend/src/scenes/session-recordings/player/eventsListLogic.test.ts [hunk 3]
```diff
@@ -117,4 +134,95 @@ describe('eventsListLogic', () => {
             expect(mockedList.scrollToPosition.mock.calls[0][0]).toEqual(40)
         })
     })
+
+    describe('eventsList', () => {
+        it('should load and parse events from the snapshot', async () => {
```
Comment (rcmarron) [inline frontend/src/scenes/session-recordings/player/eventsListLogic.test.ts:139]: naming nit: the events aren't technically from the snapshots

**Review by rcmarron — APPROVED** [review]
One small comment, but looks great!

**Pull request merged**